sqlglot/generator.py

```diff
@@ -1594,8 +1606,8 @@ def literal_sql(self, expression: exp.Literal) -> str:
 
     def escape_str(self, text: str) -> str:
         text = text.replace(self.QUOTE_END, self._escaped_quote_end)
-        if self.ESCAPE_LINE_BREAK:
-            text = text.replace("\n", "\\n")
+        if self.UNESCAPE_SEQUENCES:
+            text = text.translate(UNESCAPED_SEQUENCE_TABLE)
         elif self.pretty:
             text = text.replace("\n", self.SENTINEL_LINE_BREAK)
```

Comment:
is this section correct?

Reply:
The `SENTINEL_LINE_BREAK` one?

Reply:
yea when does this hit, just double check it

Reply:
seems necessary based on https://github.com/tobymao/sqlglot/commit/fd4dbacb3